docker/ml-testing-toolkit/test-cases/collections/tests/fx/fx_tests.json

```diff
@@ -463,7 +467,7 @@
             "FSPIOP-Source": "{$inputs.FXP_PAYER_DFSP_ID}"
           },
           "body": {
-            "commitRequestId": "{$function.generic.generateUUID}",
+            "commitRequestId": "{$prev.4.callback.body.conversionTerms.conversionId}",
```

Comment:
Should conversionTerms.conversionId be same as commitRequestId?

Reply:
conversionTerms.conversionId and commitRequestId are needed to be the same for a dependency in the sdk to retieve previous fxQuote state.